airline-core/pom.xml

```diff
@@ -27,6 +27,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.github.rvesse</groupId>
+      <artifactId>airline-backcompat-javaxinject</artifactId>
```

Comment:
Should this be a dependency of `core`? By default, I don't think you should ship the `javax.inject.Inject` annotation. Or did I misunderstand what you did?

Reply:
See other comments, this is an intentional temporary change to avoid breaking upgrading users